### ISSUE TITLE
Font Ligatures Stopped Working

### ISSUE THREAD

**Issue by sw8wm2013** [microsoft/vscode#93487]
When I opened my VS Code this morning, my font ligatures stopped working, my font in my files is not right and the font in my integrated terminal had dramatically changed. I was able to get the integrated terminal back to some sense of normalcy, but I've attempted to find the solutions posted for the font ligatures and font with no luck. My relevant config info is below:

```
{
    "workbench.colorTheme": "Night Owl Light",
    "editor.fontFamily": "Fira Code",
    "editor.fontLigatures":true,
    "editor.fontSize": 15,
    "files.autoSave": "afterDelay",
    
}
```

**Comment by TimGr** [comment]
In the "Fira Code" Wiki they suggest to wrap the font name with additional apostrophes, if something doesn't work out. Maybe it'll fix the problem.

```json
"editor.fontFamily": "'Fira Code'",
```

Link to the wiki:
https://github.com/tonsky/FiraCode/wiki/VS-Code-Instructions#manually-editing-settingsjson

**Comment by eddex** [comment]
I'm experiencing the same issue as of today using the `Fira Code` font. It seems like the font has stopped working completely (not only the ligatures).

Other font settings still work in VS Code:
- `'Times new roman', Consolas`
- `Times new roman, Consolas`

And the `Fira Code` font is correctly installed on my system, I can use it without problems (and with ligatures) in other apps.

VSCode setup:
```
Version: 1.43.2 (user setup)
Commit: 0ba0ca52957102ca3527cf479571617f0de6ed50
Date: 2020-03-24T07:38:38.248Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.17763
```

**Comment by eddex** [comment]
I just tested it on another system with the following two setups:
```
Version: 1.43.1 (system setup)
Commit: fe22a9645b44368865c0ba92e2fb881ff1afce94
Date: 2020-03-18T07:01:20.184Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362
```
```
Version: 1.43.2 (system setup)
Commit: 0ba0ca52957102ca3527cf479571617f0de6ed50
Date: 2020-03-24T07:38:38.248Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362
```

Both times it worked fine. The only difference I see is that on the system from my first comment, I was using WSL and on this system I'm not.
For the WSL setup I had the font installed in both the host and the Linux subsystem.

**Comment by alexdima** [comment]
@sw8wm2013 What OS and version of VS Code are you using?
@eddex When using  WSL + remote WSL, the window rendering VS Code executes on the Windows side, so the font needs to be installed on the Windows side. Using remote WSL would not influence this in any way, so I believe there might be something else to it.

**Comment by eddex** [comment]
> @eddex When using WSL + remote WSL, the window rendering VS Code executes on the Windows side, so the font needs to be installed on the Windows side. Using remote WSL would not influence this in any way, so I believe there might be something else to it.
If I start VS Code without WSL the problem still exists, so you're right, it has nothing to do with WSL.

**Comment by alexdima** [comment]
@eddex AFAIK the user/system VS Code installation does not influence font lookup done by Chromium. May I kindly suggest that you uninstall the font, and then follow the installation directions on the official font page and install again?

**Comment by eddex** [comment]
@alexdima thanks for the tip!
When I searched the installed fonts, I noticed, that the font was not showing up anymore. I re-installed the font and at this point a popup notified me that the font is already installed (even tho it was not listed as installed).
After installing the font again, it works as expected in VS Code.

I now installed the font for all users (`right click -> Install for all users`) becuase I expect it may have been removed by some company policy.
